_includes/footer.html

```diff
@@ -47,7 +47,8 @@ <h3>Community</h3>
           <li><a href="/support.html">Support</a></li>
           <li><a href="{{ site.data.opendds.mailing_lists_uri }}">Mailing lists</a></li>
           <li><a href="/faq.html">FAQ</a></li>
-          <li><a href="https://www.omgwiki.org/dds/">DDS Standard</a></li>
+          <li><a href="https://www.omg.org/spec/DDS/">DDS Standard</a></li>
```

Comment:
I don't think we need the spec link here, it's already on https://opendds.org/documents.